paddle/fluid/inference/CMakeLists.txt

```diff
@@ -23,17 +36,6 @@ endif()
 
 get_property(fluid_modules GLOBAL PROPERTY FLUID_MODULES)
```

Comment:
please explain why add_subdirectory(api) is in the below

Reply:
Done